src/camvidlog2/queries.py

```
@@ -66,3 +66,20 @@ def calculate_distances(
         distances = distances.droplevel(0)
 
     return distances
+
+
+def calculate_results(distances: pd.Series, num: int = 15) -> pd.DataFrame:
```

Ruff (0.11.9)

Unused function argument: `num`

(ARG001)